faw/pdf-observatory/common/dsl.ts

```diff
@@ -140,7 +153,7 @@ function generateDslParser() {
         / filters_pattern_check_expr_compare
 
       filters_pattern_check_expr_compare
-        = left:filters_pattern_check_expr_add WS* symb:(">" / ">=" / "<" / "<=" / "==") WS* right:filters_pattern_check_expr_add
+        = left:filters_pattern_check_expr_add WS* symb:(">=" / ">" / "<=" / "<" / "==") WS* right:filters_pattern_check_expr_add
```

Comment:
Unrelated fix: Due to greedy parsing, `<=`/`>=` must be before `<`/`>` to support all alternatives

Reply:
Apparently pegjs can only backtrack like that when the alternatives are at a higher level. The DSL didn't accept `>=`.

> expression1 / expression2 / ... / expressionn
> Try to match the first expression, if it does not succeed, try the second one, etc. Return the match result of the first successfully matched expression. If no expression matches, consider the match failed.